frontend/src/metabase/dashboard/components/DashCard/DashCardVisualization.tsx

```diff
@@ -254,9 +252,6 @@ export function DashCardVisualization({
         [CS.overflowAuto]: visualizationOverlay,
         [CS.overflowHidden]: !visualizationOverlay,
       })}
-      classNameWidgets={cx({
-        [cx(CS.textLight, CS.textMediumHover)]: isEmbed,
-      })}
```

Comment:
There are no other mentions of `classNameWidgets` in a repo, so it seems we can delete it.